packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/index.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
+import 'ovh-ui-angular';
+
+import choice from './telephony-line-phone-accessories-choice.component';
```

Comment:
Nitpicking: As the component is scoped, I think we can reduce the file name which will ease maintenance and readability 
```suggestion
import choice from './accessories-choice.component';
```